relay/exchange/exchange_db.py

```diff
@@ -116,6 +116,30 @@ def get_orderbook_by_tokenpair(self, token_pair: Tuple[str, str], desc_price: bo
                                 OrderORM.expiration_timestamp_in_sec))
         return [order_orm.to_order() for order_orm in orders_orm]
 
+    def get_orders(self, query: dict) -> Sequence[Order]:
+        orders_orm = self.session.query(OrderORM)
+
+        if (query['exchangeContractAddress'] is not None):
```

Comment:
this couples the parameter of this backend function too tight to the Rest API. 
Please refactor it to either have them all as separate optional parameter> (`get_orders(self, filter_exchange_address=None, filter_token_address=None...)`)
or by introducing an `OrderFilter(exchange_address, maker_token_address, ...)` class and a Marshmallow Schema to translate between the too.